moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -820,6 +820,27 @@ bool IKFastKinematicsPlugin::searchPositionIK(const geometry_msgs::Pose& ik_pose
       return false;
     }
 
+    // rotate joints by +/-360° where it is possible and useful
+    for (std::size_t i = 0; i < solutions.size(); i++)
+    {
+      for (std::size_t j = 0; j < num_joints_; ++j)
+      {
+        double distance = solutions[i][j] - ik_seed_state[j];
+        while (distance > M_PI && (!joint_has_limits_vector_[j] ||
+                                   (solutions[i][j] - 2 * M_PI > (joint_min_vector_[j] - LIMIT_TOLERANCE))))
+        {
+          distance -= 2 * M_PI;
+          solutions[i][j] -= 2 * M_PI;
+        }
+        while (distance < -M_PI && (!joint_has_limits_vector_[j] ||
+                                    (solutions[i][j] + 2 * M_PI < (joint_max_vector_[j] - LIMIT_TOLERANCE))))
```

Comment:
Should this be 
`(solutions[i][j] + 2 * M_PI < (joint_max_vector_[j] + LIMIT_TOLERANCE)` with **+** LIMIT_TOLERANCE instead of **-**?